website/content/docs/sync/index.mdx

```diff
@@ -127,6 +127,24 @@ Granularity can be updated. The new granularity only affects secrets newly assoc
 not modify the previously synced secrets. It is possible to update an association to force a recreate operation.
 The secret synced with the old granularity will be deleted and new secrets will be synced according to the new granularity.
 
+
+## Reconciliation
```

Comment:
Is it possible to point out that this is one way?

We do not detect nor import out-of-band overwrites that occur directly on secrets at the destination.